### DISCUSSION TITLE
Minicart and Cart page JS not working

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#42] [1. Help]
Hi There,

I am getting below error : Uncaught TypeError: Window.getComputedStyle: Argument 1 is not an object.

![image](https://user-images.githubusercontent.com/9527439/209843178-2ba71377-2b80-4d4f-afec-f8d5a617928d.png)

Can you please help me how to fix this error?

Thanks

**Comment by vovayatsyuk** [comment]
I can't reproduce this error. 

- Do you have the same issue at [our demo](https://breeze.swissupdemo.com/breeze_evolution/)? 
- What browser do you use?
- Can you share the link to your site?

**Comment by rutulritz** [comment]
Hi @vovayatsyuk ,

FYI, I have created theme 'AAAA' and its Parent is Breeze Evolution.
Also created another theeme 'BBBB' and I set its parent 'AAAA'. 

So is it fine by using the parent for my third theme from Breeze parent to his child theme.

It will work ?

Thanks

**Reply by rutulritz** [reply]
Can you please reply for the help as I am stuck for the development here. 

Thanks

**Reply by vovayatsyuk** [reply]
> It will work?

Yes. Breeze doesn't change anything in Magento theming possibilities.

**Reply by rutulritz** [reply]
Is the breeze is disabled for cart and checkout page by default?

**Reply by vovayatsyuk** [reply]
Yes. Breeze js is not supported on the cart and checkout pages.

**Comment by rutulritz** [comment]
How Can I make Store switcher using js trigger ?

**Reply by vovayatsyuk** [reply]
Sorry, I don’t understand your question. Just create store views using magento backend, clear cache and you’ll see the store switcher..
